e2e/Makefile

```diff
@@ -0,0 +1,8 @@
+## e2e-test: Run end to end tests via docker.
+e2e-test:
```

Comment:
IMO this should be a part of main `Makefile` (in root directory).

Reply:
moved it 🫡